### ISSUE TITLE
2.1 UX: Create Project Flow, Step 1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#284]
```
Step 1 of the new Create a Project flow guides users through getting a raw data 
file into Refine from:

- This Computer
- Google Spreadsheets
- a URL
- the clipboard

clicking "Next" here will take users to Step 2, documented in the next issue, 
where they will preview and edit import options.
```

Original issue reported on code.google.com by `jamesh...@google.com` on 15 Dec 2010 at 2:17

Attachments:
- [1-refine-import-r1.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-284/comment-0/1-refine-import-r1.png)
- [2-refine-import-r1.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-284/comment-0/2-refine-import-r1.png)
- [3-refine-import-r1.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-284/comment-0/3-refine-import-r1.png)
- [4-refine-import-r1.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-284/comment-0/4-refine-import-r1.png)
- [5-refine-import-r1.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-284/comment-0/5-refine-import-r1.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `jamesh...@google.com` on 15 Dec 2010 at 2:18
- Added labels: **Priority-Medium**
- Removed labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
Looking good !
Does the link take them to 
http://code.google.com/p/google-refine/wiki/Importers ?

```

Original comment by `thadguidry` on 15 Dec 2010 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
I'm impressed, this looks great.

When we get Fusion tables support, will we add an additional tab for that?
```

Original comment by `iainsproat` on 15 Dec 2010 at 9:56

**Comment by GoogleCodeExporter** [comment]
```
@thad: yep!

@iain: excellent! re: Fusion Tables, that sounds right.
```

Original comment by `jamesh...@google.com` on 15 Dec 2010 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
I'll see if I can start implementing this new UI over the next few weeks :)
```

Original comment by `dfhu...@gmail.com` on 23 Dec 2010 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
Revised and implemented, to be in 2.5.
```

Original comment by `dfhu...@gmail.com` on 1 Sep 2011 at 6:37
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@google.com` on 9 Oct 2011 at 5:13
- Added labels: **Milestone-2.5**
